etc/kayobe/ansible/vault-deploy-overcloud.yml

```
@@ -24,92 +24,95 @@
         name: hvac
         state: latest
         extra_args: "{% if pip_upper_constraints_file %}-c {{ pip_upper_constraints_file }}{% endif %}"
         virtualenv: "{{ virtualenv is defined | ternary(virtualenv, omit) }}"
       become: "{{ virtualenv is not defined }}"
 
     - name: Ensure /opt/kayobe/vault exists
-      file:
+      ansible.builtin.file:
         path: /opt/kayobe/vault
         state: directory
 
     - name: Template out TLS key and cert
-      copy:
+      ansible.builtin.copy:
         # Within the Hashicorp Vault container these uids & gids map to the vault user
         src: "{{ kayobe_env_config_path }}/vault/{{ item }}"
-        dest: "/opt/kayobe/vault/{{ item }}"
+        dest: /opt/kayobe/vault/{{ item }}
         owner: 100
         group: 1000
-        mode: 0600
+        mode: "0600"
       loop:
         - "{% if kolla_internal_fqdn != kolla_internal_vip_address %}{{ kolla_internal_fqdn }}{% else %}overcloud{% endif %}.crt"
         - "{% if kolla_internal_fqdn != kolla_internal_vip_address %}{{ kolla_internal_fqdn }}{% else %}overcloud{% endif %}.key"
-      become: True
+      become: true
 
-    - import_role:
+    - name: Apply vault role
+      ansible.builtin.import_role:
         name: stackhpc.hashicorp.vault
       vars:
         hashicorp_registry_url: "{{ overcloud_hashicorp_registry_url }}"
         hashicorp_registry_username: "{{ overcloud_hashicorp_registry_username }}"
         hashicorp_registry_password: "{{ overcloud_hashicorp_registry_password }}"
         consul_docker_image: "{{ overcloud_consul_docker_image }}"
         consul_docker_tag: "{{ overcloud_consul_docker_tag }}"
-        vault_config_dir: "/opt/kayobe/vault"
-        vault_cluster_name: "overcloud"
+        vault_config_dir: /opt/kayobe/vault
+        vault_cluster_name: overcloud
         vault_ca_cert: "{{ '/etc/pki/tls/certs/ca-bundle.crt' if ansible_facts.os_family == 'RedHat' else '/usr/local/share/ca-certificates/OS-TLS-ROOT.crt' }}"
         vault_docker_image: "{{ overcloud_vault_docker_image }}"
         vault_docker_tag: "{{ overcloud_vault_docker_tag }}"
         vault_tls_cert: "{% if kolla_internal_fqdn != kolla_internal_vip_address %}{{ kolla_internal_fqdn }}{% else %}overcloud{% endif %}.crt"
         vault_tls_key: "{% if kolla_internal_fqdn != kolla_internal_vip_address %}{{ kolla_internal_fqdn }}{% else %}overcloud{% endif %}.key"
         copy_self_signed_ca: true
-        vault_api_addr: "https://{{ internal_net_name | net_ip }}:8200"
+        vault_api_addr: https://{{ internal_net_name | net_ip }}:8200
         vault_write_keys_file: true
         vault_write_keys_file_path: "{{ kayobe_env_config_path }}/vault/overcloud-vault-keys.json"
 
     - name: Include Vault keys
-      include_vars:
+      ansible.builtin.include_vars:
         file: "{{ kayobe_env_config_path }}/vault/overcloud-vault-keys.json"
         name: vault_keys
 
-    - import_role:
+    - name: Import vault unseal role
```

```suggestion
    - name: Apply vault unseal role
```

Let's keep this consistent with all the other vault playbooks